docs/workshops2019/index.md

```diff
@@ -0,0 +1,34 @@
+# Workshop Series 2019
+
+
+## Source Control with Git
+
+
+This workshop goes over the basics of source control and how git can be used to effectively manage project versioning.
+
+GitPresentation.pptx is the intro slideshow that goes over the basics of source control and git.
+
+demonotes.md are the speaker notes used throughout the demo including helpful learning resources at the end. The speaker notes also include all code that was written throughout the demo.
+
+[Source Control with Git](git/index.md)
```

Comment:
Prob should reformat this and reword to be used in Jumpstart. Also we should add more about git documented